lib/registries/addon/components/registries-layout/component.ts

```diff
@@ -0,0 +1,52 @@
+import { tagName } from '@ember-decorators/component';
+import { action, computed } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import Media from 'ember-responsive';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import defaultTo from 'ember-osf-web/utils/default-to';
+
+import styles from './styles';
+import template from './template';
+
+@tagName('')
+@layout(template, styles)
+export default class RegistriesLayout extends Component {
```

Comment:
We'll soon (🤞) be using this layout on non-registries pages, could you move it to `osf-components` and rename it to something less specific? Maybe just `Layout`, or `OsfLayout`?